infrastructure/application/utils/generateCORSAllowList.ts

```diff
@@ -7,6 +7,8 @@ export const generateCORSAllowList = (customDomains: CustomDomains, domain: stri
   const editorURL = `https://${domain}`;
   const apiURL = `https://api.${domain}`; // Required for requests from API docs
   const hasuraURL = `https://hasura.${domain}`; // Required for proxied auth requests
+  // TODO: Configure when staging / production infra for LPS is put in place
+  // const lpsURL = "https://localplanning.services"
```

Comment:
Intentionally left as a `TODO`, updated https://github.com/orgs/theopensystemslab/projects/6?pane=issue&itemId=112367810 with a note on this.